raven/src/main/java/com/getsentry/raven/RavenFactory.java

```diff
@@ -109,6 +109,20 @@ public static Raven ravenInstance(Dsn dsn, String ravenFactoryName) {
             }
         }
 
+        if (ravenFactoryName != null && triedFactories.size() < 1) {
```

Comment:
I mistakenly started with `> 1` and just edited, but it should be `isEmpty()` anyway, fixing.